package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "OutboundFundsNPSP",
+  "license": "UNLICENSED",
+  "author": "Salesforce.org Impact Engineering",
+  "devDependencies": {
+    "@lwc/eslint-plugin-lwc": "^0.10.0",
+    "@lwc/jest-preset": "^7.0.0",
+    "@sa11y/jest": "^0.2.0",
+    "@salesforce/eslint-config-lwc": "^0.6.0",
+    "@salesforce/lwc-jest": "^0.6.2",
```

Comment:
Is this the right version of lwc-jest for API 50? I thought this was older but I may be misremembering.

Reply:
It seems there was an error creating new versions of `@salesforce/sfdx-lwc-jest`.   API Version 50.0 uses `0.11.0` while API Version 51.0 is on `0.10.x`.   Updating.